inst/extdata/PA2022CH_en_exec_summary/survey-chapter.Rmd

```diff
@@ -27,88 +20,60 @@ knitr::include_graphics(file.path(survey_dir, language, "plot_layer_with_peer_in
 Research shows that engagement is one of the most effective ways to improve climate goal alignment of companies invested through equity and bonds. Therefore, the engagement dart charts below show your engagement measures for three climate crucial sectors and thereby complement the previous alignment section. The text besides provides a comparison to the share of peers with best practice, i.e., the ones that have all layers covered.
 
 
-```{r plot_layer_with_peer_info_engagement_Fossil_Fuels, fig.height = 6}
-# the specific PACTA outputs that we need here are my guesses 
-knitr::include_graphics(file.path(survey_dir, language, "plot_layer_with_peer_info_engagement_Fossil_Fuels.png"))
-# TODO: add title - Power engagement badge.
-```
-```{r plot_layer_with_peer_info_engagement_Automotive, fig.height = 6}
-# the specific PACTA outputs that we need here are my guesses 
-knitr::include_graphics(file.path(survey_dir, language, "plot_layer_with_peer_info_engagement_Automotive.png"))
-# TODO: add title - Automotive engagement badge.
-```
-```{r plot_layer_with_peer_info_engagement_Power, fig.height = 6}
-# the specific PACTA outputs that we need here are my guesses 
-knitr::include_graphics(file.path(survey_dir, language, "plot_layer_with_peer_info_engagement_Power.png"))
-# TODO: add title - Fossil fuels engagement badge.
+```{r plot_layer_with_peer_info_engagement, out.height='30%', out.width='33%', fig.show='hold', fig.cap = 'Engagement badges and peer comparison per sector. Left: Fossil Fuels, Middle: Automotive, Right: Power.'}
+knitr::include_graphics(c(file.path(survey_dir, language, "plot_layer_with_peer_info_engagement_Fossil_Fuels.png"),
+file.path(survey_dir, language, "plot_layer_with_peer_info_engagement_Automotive.png"),
+file.path(survey_dir, language, "plot_layer_with_peer_info_engagement_Power.png")))
```

Comment:
I don't know how this stuff works, but it seems odd to me that the Fossil Fuels image appears to be added with `knitr::include_graphics()` while the others just use `file.path()`

Reply:
it's all part of ine big `knitr::include_graphics()` call.. Maybe not the best formatting on my side ;)

Reply:
maybe some indentation would make that more clear?

Reply:
👍🏻